services/monitor/src/ChainMonitor.ts

```diff
@@ -261,7 +264,7 @@ export default class ChainMonitor extends EventEmitter {
         return;
       }
       try {
-        const cborData = bytecodeDecode(bytecode);
+        const cborData = bytecodeDecode(bytecode, AuxdataStyle.SOLIDITY);
```

Comment:
Same as in `session-state.handler.ts`. Just add a comment that we decode it with Solidity style because it is the only published to ipfs.